src/webgl/p5.RendererGL.js

```diff
@@ -1755,18 +1757,16 @@ p5.RendererGL = class RendererGL extends Renderer {
     }
 
     const fill = this.userFillShader;
+    if(fill){
+      return fill;
+    }
     if (this._enableLighting) {
-      if (!fill || !fill.isLightShader()) {
-        return this._getLightShader();
-      }
-    } else if (this._tex) {
-      if (!fill || !fill.isTextureShader()) {
-        return this._getLightShader();
-      }
-    } else if (!fill /* || !fill.isColorShader()*/) {
-      return this._getColorShader();
+      return this._getLightShader();
     }
-    return fill;
+    if (this._tex) {
```

Comment:
Can we use the OR operator here and combine it into a single if statement? Something like:
```
if(this._enableLighting || this._tex) 
return this._getLightShader();
```
Can this make the code cleaner instead of using two separate if blocks?